ansible/roles/ocp-workload-istio-prometheus-demo/tasks/workload.yml

```diff
@@ -81,10 +93,22 @@
 
 ########## Setup Hello-World Projects ############
 - name: Create Hello World CI/CD Project
-  shell: |
-    oc new-project {{ocp_project_name}}-cicd --display-name='{{ocp_project_display_name}} CI/CD'
-    oc process -f https://raw.githubusercontent.com/{{git_user[0]}}/{{git_repo_short[0]}}/{{git_branch}}/k8s/production/cicd-project-template.yaml -p SOURCE_REPOSITORY_URL={{git_repo}} -p BRANCH={{git_branch}} -p APP_SUBDOMAIN={{ocp_apps_domain}} | oc create -f-
-    oc set resources deploymentconfig jenkins --limits=cpu=1500m,memory=1Gi
+  shell: >-
```

Comment:
This should be cleaned up for readability.

Reply:
I cleaned it up in the way I thought was most readable, because of the long url. Please give me a suggestion indicating how you'd prefer it so I can change it.